docs/api-reference/self-signed-certificates.md

```diff
@@ -0,0 +1,61 @@
+<!--pytest-codeblocks:skipfile-->
+<!-- markdownlint-disable-next-line first-line-h1 -->
+When using the `quilt3` API or CLI with a client-to-site (remote
+access) VPN you may need to use a self-signed certificate to
+securely connect. This requires adding the certificate to your
+Python environment.
+
+## Mac OS X
+
+1. Open Keychain Access.
+2. Select System Keychains > System Roots.
+3. Select the «Certificates» tab.
+4. Select a root certificate (e.g. AAA Certificate Services).
```

Comment:
Should probably make this consistent with the Windows instructions: start with open a browser, go to a website, get the certificate issuer, etc. - and then in this step, look for the certificate from earlier.